### ISSUE TITLE
"Native module cannot be null" thrown when importing Purchases in javascript

### ISSUE THREAD

**Issue by squgeim** [RevenueCat/react-native-purchases#23]
I have followed the installation instructions and got the project to build successfully. Now, when I import Purchases in my javascript code I get the exception "Native module cannot be null".

![screen shot 2018-10-04 at 18 49 30](https://user-images.githubusercontent.com/4996818/46475797-3cc98980-c806-11e8-8ef0-51193e9b288c.png)

Where `revenueCatService.js:2` is where I have imported Purchases.

Clicking on `<unknown> index.js:5:21` takes me to `index.js` in react-native-purchases:

![screen shot 2018-10-04 at 18 50 39](https://user-images.githubusercontent.com/4996818/46475872-6682b080-c806-11e8-8d51-8959c3ba8676.png)

**System**
Xcode 10
MacOS Mojave
React native 0.50.4

**Comment by squgeim** [comment]
I was able to solve the issue by adding `libRNPurchases.a` in the `Linked Frameworks and Libraries`.

**Comment by systemride** [comment]
@vegaro This is a critical step that is missing from the README. I was getting the same error until I did the step above of adding `libRNPurchases.a` to `Link Binary with Libraries`. You might want to add this as one of the gifs.

**Comment by jeiting** [comment]
Hey @systemride,

Did you run `react-native link react-native-purchases`?

That should make sure the RNPurchases lib file is integrated.

**Comment by systemride** [comment]
When I run `react-native link react-native-purchases` all that does is add a line to my `Podfile`. I am unable to use cocoapods for this so I add it in manually.

**Comment by jeiting** [comment]
Got it. Thanks.

We'll run through the integration again cold and make sure we get the
missing step added.
ᐧ

**Comment by vegaro** [comment]
I added the step to the README. I was able to reproduce this by running `react-native link` in a project that contains a Podfile. It looks like `react-native link` behaves in a different way if it detects the project uses Cocoapods. If the project doesn't use Cocoapods (there is a Podfile) the link to the static library is properly added.

**Comment by jwknz** [comment]
Hi, I have followed the instructions a lot and `libRNPurchases.a` is not showing up for me - I have cleared my caches and reinstalled the packages so many times I have lost count 😄 

Is there a place where I can download `libRNPurchases.a` and add it manually? I am at a loss for what to do next:

This is my react-native info output:

```
System:
    OS: macOS 10.15
    CPU: (4) x64 Intel(R) Core(TM) i5-8210Y CPU @ 1.60GHz
    Memory: 228.49 MB / 16.00 GB
    Shell: 3.2.57 - /bin/bash
  Binaries:
    Node: 10.16.3 - ~/.nvm/versions/node/v10.16.3/bin/node
    Yarn: 1.19.0 - ~/.yarn/bin/yarn
    npm: 6.9.0 - ~/.nvm/versions/node/v10.16.3/bin/npm
  SDKs:
    iOS SDK:
      Platforms: iOS 13.0, DriverKit 19.0, macOS 10.15, tvOS 13.0, watchOS 6.0
  IDEs:
    Android Studio: 3.5 AI-191.8026.42.35.5791312
    Xcode: 11.0/11A420a - /usr/bin/xcodebuild
  npmPackages:
    react: 16.8.6 => 16.8.6 
    react-native: ^0.59.8 => 0.59.10 
  npmGlobalPackages:
    react-native-cli: 2.0.1
    react-native-git-upgrade: 0.2.7
    react-native: 0.61.1
info React Native v0.60.0 is now available (your project is running on v0.59.10).
info Changelog: https://github.com/facebook/react-native/releases/tag/v0.60.0.
info Diff: https://react-native-community.github.io/upgrade-helper/?from=0.59.10.
info To upgrade, run "react-native upgrade".
```

I am running Catalina, but other than that I am not using beta software - react native is still on 0.59.10 for the project and Xcode 11 has been released.
